src/state/transactions/actions.ts

```diff
@@ -18,13 +18,24 @@ export const addTransaction = createAction<{
     from: string
     approval?: { tokenAddress: string; spender: string }
     claim?: { recipient: string }
-    summary?: string
+    summary?: string,
+    tradeInfo?: {
+      input: {
+        decimals: number | undefined,
+        symbol: string | undefined
+      },
+      output: {
+        decimals: number | undefined,
+        symbol: string | undefined
+      }
+    }
```

Comment:
why is tradeinfo required? isnt summary enough?

Reply:
So in short: The summary for pending and confirmed is being updated in the process

I decided to add this because the pending summary (as it is now) is just plain string.
I needed the tradeInfo for building up the confirmed summary in the updater
And it felt easier by adding it here, than to decode the TX receipt to get the info

I tried using Currency first, but I couldn't figure out the right way to do it, as I got type issues because I didn't know how to get from uni-sdk Currency (buy/sell) to sushi-sdk Currency (SwapConfirmModal)